app/models/session.rb

```diff
@@ -50,6 +50,7 @@ def setup_defaults
   def within_maximum_ttl
     time_remaining = (@created_at + MAX_SESSION_LIFETIME - Time.now.utc).round
     if time_remaining.negative?
+      StatsD.increment('api.session.max_duration', tags: ["uuid:#{self.class.obscure_token(@uuid)}"])
```

Comment:
do you need to initialize this?

Reply:
This'll probably cause a huge increase in series that prometheus can't handle, but maybe we can do without the tag?. pinging @wyattwalter for confirmation.

Reply:
> This'll probably cause a huge increase in series that prometheus can't handle

I see.  The reason for the tag is because 1 user session may trigger this `StatsD.increment` multiple times, but I only want to count it once.  I'm interested in knowing how many individual user sessions reach the max duration.  I expect this to be _very_ low since the max session duration is `12 hours`, but I can understand the concern since the number of unique `uuid`'s we have is high.

On a side note I actually want the tag to be a hash of `uuid`+`token` since one user, but two sessions should be counted twice.

Reply:
> do you need to initialize this?

I don't _think_ so? The possible number of `tags` is infinite, so I don't see how it can be initialized to zero.